requirements-dev.txt

```diff
@@ -0,0 +1,2 @@
+pip
+pytest-cov
```

Comment:
`conda-forge` does not have `coveralls` yet so that is the only one I am installing with `pip` in `.travis.yml`.